src/constants/sfx.asm

```diff
@@ -1,6 +1,8 @@
 ;
 ; Constants for audio effects
-;
+
+; @TODO Are these correct? Some of these don't seem to match
+; their actual in-game use (???)
```

Comment:
Yeah, these are probably not correct. When @Drenn1 disassembled the music code, he also found that this list wasn't matching the music tracks.

I found this list in LALE reverse-engineering notes, I think, but obviously it doesn't refer to the same think. I opened an issue to fix this (#221)